dask_kubernetes/operator/deployment/helm/dask-kubernetes-operator/crds/kopfpeering.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterkopfpeerings.kopf.dev
+spec:
+  scope: Cluster
+  group: kopf.dev
+  names:
+    kind: ClusterKopfPeering
+    plural: clusterkopfpeerings
+    singular: clusterkopfpeering
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            status:
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: kopfpeerings.kopf.dev
+spec:
+  scope: Namespaced
+  group: kopf.dev
+  names:
+    kind: KopfPeering
+    plural: kopfpeerings
+    singular: kopfpeering
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            status:
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
+---
+apiVersion: kopf.dev/v1
+kind: ClusterKopfPeering
+metadata:
+  name: default
```

Comment:
The name here needs to match the name you set in the operator config -> `dask-kubernetes-operator`